### ISSUE TITLE
deCONZ GUI crashes on `float` value

### ISSUE THREAD

**Issue by ebaauw** [dresden-elektronik/deconz-rest-plugin#2379]
See https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1654#issuecomment-578460731.

The _Attribute Editor_ window doesn't show an input field for a `float` value.
When pressing _Read_, deCONZ crashes.
![Screenshot 2020-01-26 at 12 26](https://user-images.githubusercontent.com/22179355/73134468-2e286680-4037-11ea-9bfd-e39f6bf57ab0.png)

**Comment by manup** [comment]
Hmm strange, not much special code wise. Perhaps the second line makes the trouble.

The U32 and float are in a C union.

![image](https://user-images.githubusercontent.com/383386/73160030-3be9f480-40e8-11ea-81e0-891150290fab.png)

Is the debug print visible before crash?

**Comment by ebaauw** [comment]
I think it did.  Took deCONZ long enough to crash after that, though.
```
Jan 27 19:38:28 pi2 deCONZ[3030]: 19:38:28:138 float: 0x00000000
...
Jan 27 19:38:36 pi2 deCONZ[3030]: 19:38:36:656 Poll light node Terrace 2
Jan 27 19:38:39 pi2 systemd[1]: deconz-gui.service: Main process exited, code=dumped, status=11/SEGV
Jan 27 19:38:46 pi2 systemd[1]: deconz-gui.service: Unit entered failed state.
Jan 27 19:38:46 pi2 systemd[1]: deconz-gui.service: Failed with result 'core-dump'.
```
On second and third attempt, I didn't see the message, though, and deCONZ crashed quicker after pressing _Read_.

Core dump (of first attempt): https://app.box.com/s/sa0qwgh5l6iibhded5o4wan6luq3w1wi.

**Comment by manup** [comment]
Thanks for the core dump, this helped. I've added the missing pieces in the attribute editor to show and write float values in various formats like 1000, 1000.0 and 1e3. Will be available in 2.05.73.

![image](https://user-images.githubusercontent.com/383386/73539567-f43bd380-442d-11ea-96aa-0c1b1babbdb7.png)

**Comment by ebaauw** [comment]
Thanks.  You might want to check the spelling of precision.  Or is this taken from general.xml?

**Comment by ebaauw** [comment]
> Will be available in 2.05.73.

I can now read the attribute normally.  Thanks.
